modules/402-ingress-nginx/crds/doc-ru-ingress-nginx.yaml

```
@@ -491,6 +491,9 @@ spec:
                         Заголовок, из которого будет получен настоящий IP-адрес клиента.
 
                         > Работает только при включении `behindL7Proxy`.
+                    trustedCIDRs:
```

```suggestion
                    acceptClientIPHeadersFrom:
```
trustedCIDRs conflicts with acceptRequestsFrom.

---

renamed to acceptClientIPHeadersFrom